docs/sources/trackers/google-tag-manager/settings-template/index.md

```diff
@@ -0,0 +1,150 @@
+
+## Predefined Contexts
+
+Predefined contexts provide additional metadata for your Snowplow events. By including these contexts, you can capture common data points like device information, session details, or geolocation without having to define them manually.
+
+Available predefined contexts are:
+
+| Name                          | Description                                                                                                                              | Schema                                                                                                                                                    |
+| ----------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `webPage`                     | Information about the web page where the event occurred.                                                                                 | [Web Page context](https://github.com/snowplow/iglu-central/blob/master/schemas/org.schema/WebPage/jsonschema/1-0-0)                                      |
+| `gaCookies`                   | Information about the Google Analytics cookies.                                                                                          | [Google Analytics Cookies context](https://github.com/snowplow/iglu-central/blob/master/schemas/com.google.ga4/cookies/jsonschema/1-0-0 )                 |
+| `clientHints`                 | Information about the client's device.                                                                                                   | [Client Hints context](https://github.com/snowplow/iglu-central/blob/master/schemas/org.ietf/http_client_hints/jsonschema/1-0-0)                          |
+| `geolocation`                 | Information about the client's geolocation.                                                                                              | [Geolocation context](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/geolocation_context/jsonschema/1-1-0)   |
+| `session`                     | Information about the user session.                                                                                                      | [Session context](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/client_session/jsonschema/1-0-2)            |
+| `performanceNavigationTiming` | Retrieves data from the [PerformanceNavigationTiming](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming) API. | [Performance Navigation Timing context](https://github.com/snowplow/iglu-central/tree/master/schemas/org.w3/PerformanceNavigationTiming/jsonschema/1-0-0) |
```

Comment:
Linking to the responsible plugins might be more useful than the schemas, which should be explained there anyway.